drivers/ethernet/phy/phy_tja1103.c

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_tja1103_phy
+
+#include <errno.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/net/phy.h>
+#include <zephyr/net/mii.h>
+#include <zephyr/net/mdio.h>
+#include <zephyr/drivers/mdio.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(phy_tja1103, CONFIG_PHY_LOG_LEVEL);
+
+/* PHYs out of reset check retry delay */
+#define TJA1103_PHY_AWAIT_DELAY_POLL_US			15000U
```

Comment:
nit: regardless taste and preferences, the indentation in this file is a bit off. Tab width should be set to 8 characters and max line length is 100 characters (see https://github.com/zephyrproject-rtos/zephyr/blob/main/.clang-format). I'd suggest to run `clang-format -i drivers/ethernet/phy/phy_tja1103.c` to format it following the project coding style.

Reply:
Ran and updated.